Quantum-Control-Applications/Superconducting/Single-Fixed-Transmon/02_raw_adc_traces_mw_fem.py

```diff
@@ -46,15 +46,21 @@
 ###########################
 # Run or Simulate Program #
 ###########################
-simulate = False
+simulate = True
 
```

Comment:
Need to revert this

Reply:
Weird - I see `simulate = True`.

Reply:
Yes and we want False by default

Reply:
Oh I thought you mean the other way around. Ok yes I will fix it.